test/functional/search/content-check/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
This "content-check" test is not run. Should it be added to the functional-tests script?